feeds/src/state/ducks/aggregator/reducers.ts

```diff
@@ -24,7 +24,7 @@ export const INITIAL_STATE: State = {
   latestCompletedAnswerId: null,
   pendingAnswerId: null,
   nextAnswerId: null,
-  oracleAnswers: null,
+  oracleAnswers: [],
```

Comment:
Need to change the interface `State` so that `null` isn't part of the union anymore